core/src/main/java/org/frankframework/core/Adapter.java

```diff
@@ -196,12 +196,8 @@ public void configure() throws ConfigurationException { //TODO check if we shoul
 
 		List<String> hrs = new ArrayList<>();
 		for (IPipe pipe : pipeline.getPipes()) {
-			if (pipe instanceof AbstractPipe aPipe) {
-				if (StringUtils.isNotEmpty(aPipe.getHideRegex())) {
-					if (!hrs.contains(aPipe.getHideRegex())) {
-						hrs.add(aPipe.getHideRegex());
-					}
-				}
+			if (pipe instanceof AbstractPipe aPipe && StringUtils.isNotEmpty(aPipe.getHideRegex()) && !hrs.contains(aPipe.getHideRegex())) {
```

Comment:
- Misschien beter om property `hideRegex` toe te voegen aan `IPipe`?
- Misschien beter om van `hrs` een `Set<String>` te maken ipv een `List<String>`? Dan hoeven we niet iedere keer die `contains` check te doen.